synapse/groups/groups_server.py

```diff
@@ -407,6 +407,11 @@ def get_group_profile(self, group_id, requester_user_id):
         group_description = yield self.store.get_group(group_id)
 
         if group_description:
+            join_policy = group_description['join_policy']
+            del group_description['join_policy']
+
+            group_description['is_openly_joinable'] = join_policy == "open"
```

Comment:
If we're doing this then lets be explicit about what we're returning, e.g.:

```json
group_description = {
    "name": group_description['name'],
    ...
}
```

Mainly to stop us from running into the same problem when adding columns to the DB in future

Reply:
yep, wfm